### ISSUE TITLE
layers simultaneas

### ISSUE THREAD

**Issue by luisfelipebr** [digital-guard/stats#3]
> minha sugestão para relatórios futuros é conferir se precisa excluir caso de "layers simultâneos"... Apesar de raros, existem casos como o de MG-Contagem, que estaria sendo duplicado já que tem geoaddress e parcel (precisa escolher um ou outro).

Na contagem do total de endereços recebidos, estamos contando alguns casos duas vezes: quando recebemos ambos geoaddress e parcel. O ideal seria contar apenas uma vez.

No caso de Manaus, recebemos uma camada de geoaddress incompleta e parcel completa. Uma possível regra seria contar apenas a camada com o maior número de endereços

<img width="540" alt="Screen Shot 2022-06-28 at 12 01 34" src="https://user-images.githubusercontent.com/45462352/176229426-35d6a31c-23c5-40de-8bc0-3b7fd35eb718.png">

**Comment by ppKrauss** [comment]
Conferir se as novas fontes de dados resolvem o problema da contagem dobrada, ver links em https://github.com/digital-guard/preserv/issues/113#issuecomment-1181961630

**Comment by 0e1** [comment]
> Conferir se as novas fontes de dados resolvem o problema da contagem dobrada, ver links em [digital-guard/preserv#113 (comment)](https://github.com/digital-guard/preserv/issues/113#issuecomment-1181961630)

Elas **não** consideram essa possibilidade.

Em essência, qualquer regra deve ser aplicada apenas na query da api do _tabelão_ (https://osm.codes/_sql/stats_donated_packcomponent).

Então, para considerarem, definam a(s) regra(s) para escolher entre _geoaddress_ e _parcel_ quando os dois existirem.

**Comment by ppKrauss** [comment]
A regra poderia ser "Quando tem geoaddress desconsidera parcel", mas e se tem mais lotes do que pontos?  É preciso talvez um flag (no *make_conf*)  indicando qual layer contar (e se não tem flag usa a regra).

Nota. Outra abordagem, quando for tudo dinâmico no site, é dar a opção de analisar só um layer: totais por geoaddress, totais por parcel, etc.

**Comment by ppKrauss** [comment]
> No caso de Manaus, recebemos uma camada de geoaddress incompleta e parcel completa. Uma possível regra seria contar apenas a camada com o maior número de endereços

Sim, se for simples fazer isso, pode ir tocando assim: @0e1 tudo bem, facil implementar? O novo desafio serão as views, e relatórios, que precisam ser todos amarrados a essa contagem.
